server/requirements.txt

```diff
@@ -7,3 +7,4 @@ gunicorn
 humanize
 pyesbulk>=2.0.1
 requests
+libselinux
```

Comment:
I think this should be `selinux`; at least, `pip` sees that module, and it can be imported, whereas attempting to `pip install libselinux` locally gives me the same problem as the Jenkins tests.

Reply:
yes, I changed that